### DISCUSSION TITLE
SpawnWithID="" retriggers every round. Allows for Item Duplication

### DISCUSSION THREAD

**Discussion by UnhappyWithMyUsername** [FakeFishGames/Barotrauma#14634] [Ideas and suggestions]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

When adding SpawnWithID="ItemIdentifier" to an ItemContainer, the spawned item will respawn into the container again after a loading screen if you picked it out of the container, allowing to duplicate the item with each round by simply by making sure the slot is empty while entering a loading screen.

Perhaps make it toggleable to either have it be a one-time only spawn for the item or to keep it as is in case a mod relies on this behavior.

Some examples are in the following mods:
Hungry Europans: Popped Pop Nut (respawns its seed in its inventory)
Immersive Repairs: Engines (respawns engine oil)

### Reproduction steps

1. Define SpawnWithID="YourItemIdentifier" in a container.
2. Start the game and collect the item from the container.
3. Load into next round and watch the item be refilled in the container, essentially allowing you to duplicate as much as you want by simply going through loading screens.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.5.9.2 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
This isn't actually a bug, it works intentionally like this and is intended for items that should always have some other type of item inside. I agree it would be useful to make it only spawn the item once though, I think we might look into it at some point!

**Reply by UnhappyWithMyUsername** [reply]
Ah I see! Thank you for clarifying and it is very much appreciated if someone is at some point looking into also allowing it to only spawn desired items once!
